polkadot-protocol/protocol-components/accounts/address-formats.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Address Formats
+description: This page provides an overview of Substrate address formats, encoding strategies use and best practices. 
+---
+
+# Address Formats
+
+!!! warning "Update Notice"
+
+    Please read Substrate to [Polkadot SDK](https://docs.substrate.io/polkadot-sdk/) page first.
+
+The default Substrate address format is `SS58`, a modified version of Bitcoin's `Base58Check` format tailored for Substrate-based chains. 
```

Comment:
Re: Renaming Substrate to Polkadot SDK, or add a note signifying that Substrate is part of the Polkadot SDK

Reply:
Changed the name to Substrate SDK instead of a note. It seems like a cleaner way to correct this